src/overlay.py

```diff
@@ -141,16 +183,15 @@ def __format_line(
         Returns:
             str: Attribute display name and value
         """
-        if not attribute_name in obj.__dict__:
-            return f"Ovrl. Err.: attribute {attribute_name} not found"
+
+        display_value = self.__get_display_value_from_object(obj, attribute_name)
+
```

Comment:
maybe eliminate multiple return statements with structure from comment above

Reply:
```
        display_value = self.__get_display_value_from_object(obj, attribute_name)
        formated_value = None

        if attribute_format_as == float:
            formated_value = round(display_value, 2)
        elif attribute_format_as == int:
            formated_value = int(display_value)
        elif attribute_format_as == str:
            formated_value = str(display_value)

        return f"{attribute_display_name}: {formated_value}"
```

Reply:
good! btw you do not have to repost it here in the comments, I see it in the diff view anyway